cpp/src/column/column_device_view.cu

```diff
@@ -146,11 +161,13 @@ std::unique_ptr<mutable_column_device_view, std::function<void(mutable_column_de
   return p;
 }
 
-size_type mutable_column_device_view::extent(mutable_column_view source) {
-  size_type data_size = sizeof(mutable_column_device_view);
-  for( size_type idx=0; idx < source.num_children(); ++idx )
-    data_size += extent(source.child(idx));
-  return data_size;
+std::size_t mutable_column_device_view::extent(mutable_column_view source) {
+  auto get_extent = thrust::make_transform_iterator(
+      thrust::make_constant_iterator(0),
```

Comment:
Should this be a counting iterator? Seems weird to always use child 0.

Reply:
Ho boy, good catch. Auto-complete did me wrong.